internal/datastore/spanner/migrations/zz_migration.0007_register_combined_change_stream.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+var createCombinedChangeStream = `CREATE CHANGE STREAM combined_change_stream FOR namespace_config,caveat,relation_tuple`
```

Comment:
🤔 we are adding a new change stream, but we already had other 2. So this by itself is not a breaking change, but at some point we will have to drop the old ones, which makes this a multi-phase migration.

Imagine this change drops in SpiceDB 1.29, and then the old change streams are dropped in 1.30.
The operator can jump directly to 1.30 if needed, will run creating the combined, then dropping, and then will roll SpiceDB 1.30. While 1.30 is rolling, the previous deployment may see their Watch API and experimental schema cache unavailable.

@ecordell can you confirm?

Reply:
Or we just don't drop the old ones

Reply:
We can encode the "phases" at any point in the update graph that we want. In your example we would mark 1.29 as a "way point" in the update graph that all spanner users have to stop through before going further. 

That said, some of our release automation assumes that phased migrations will happen within the same version. So unless we have a good reason to not do this now, it's easier all around (and harder to forget) to just configure the flags for this release.

Reply:
@josephschorr ☝🏻 thoughts? Please note that GCP charges for Spanner's storage usage (data and backups), including what's stored on Change Streams. They also write the change streams in the same transaction as the originating mutation, but they don't clarify how much of a latency impact it has.

Reply:
*sigh* I guess we'll do it now then; I don't want to block the PR but such is life